src/renderer/lib/telemetry.js

```diff
@@ -55,41 +73,6 @@ function reset () {
   }
 }
 
-function postToServer () {
```

Comment:
I can require the module with a different name, or use the `.post()` helper method. 

`get.post()` is equivalent to `get({method: 'POST'})`